### DISCUSSION TITLE
Can't use the plugin, missing modules error on Blueprint Project.

### DISCUSSION THREAD

**Discussion by aoshikearun** [Sixze/ALS-Refactored#117] [Q&A] [answered]
I tried adding it to the *.uproject file, recompiling the whole solution, not working.
I still get the infamous "The following modules are missing or built with a different engine version" message.

**Comment by Sixze** [comment, marked as answer]
I don't quite understand what you are trying to do. Currently blueprint-only projects are not supported, at least I haven't tested it.

**Reply by aoshikearun** [reply]
Well, I'm just trying to install the plugin copying to the folder and recompiling the project, but I get the modules missing message as soon as I open the project (I also tested by editing the *.uproject file as text to add the modules manually, with no avail). However, as you stated, it might just be incompatible with blueprint-only projects.
